libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -158,6 +158,18 @@ void SRV_Channels::setup_failsafe_trim_all(void)
     }
 }
 
+// get pwm output for all channels
+bool SRV_Channels::get_all_outputs_pwm(uint16_t values[], uint8_t chan_num){
```

Comment:
While I don't object to this helper being added it has a risk in that if we ever extend NUM_SERVO_CHANNELS past 16 it will cause the Volz protocol handler to not realize it, and corrupt memory. Given that you are iterating with a bitmask anyways for which channels to send would it be insufficent to simply request each channel one at a time?

Reply:
the main reason for the helper method is from gitter :)
"Andrew Tridgell @tridge Oct 18 13:21
I think you'd make SRV_Channel push you an array of 16 values, instead of pulling"

great inputs, will make the changes today.

Reply:
now pulling in one time all data for all channels.